src/components/Download/DatasetDetails/ExperimentView/index.js

```diff
@@ -1,43 +1,36 @@
 import { useState } from 'react'
 import { Paragraph } from 'grommet'
 import { useDatasetManager } from 'hooks/useDatasetManager'
-import getReadableOptions from 'helpers/getReadableOptions'
+import getFormattedExperiments from 'helpers/getFormattedExperiments'
 import { OrganismFilter } from './OrganismFilter'
 import { ViewBlock } from './ViewBlock'
 import { ViewBlocks } from '../ViewBlocks'
 
-export const ExperimentView = ({
-  dataset: {
-    data: datasetData,
-    id: datasetId,
-    experiments,
-    quantile_normalize: quantileNormalize
-  },
-  isImmutable
-}) => {
+export const ExperimentView = ({ dataset, isImmutable }) => {
   const { formatSampleMetadata } = useDatasetManager()
+  const defaultOrganismFilterOption = { label: 'All Speciess', value: 'ALL' }
   const defaultValue = 'ALL'
-  const defaultOrganismFilterOption = getReadableOptions([defaultValue])[0]
-  const [organism, setOrganism] = useState(defaultValue)
+  const formattedExperiments = getFormattedExperiments(dataset)
+  const [organism, setOrganism] = useState(defaultOrganismFilterOption.value)
 
-  if (!datasetData || !Object.keys(datasetData).length) {
+  if (!dataset.data || !Object.keys(dataset.data).length) {
     return <Paragraph>No samples added to download dataset.</Paragraph>
   }
 
   return (
     <>
       <OrganismFilter
-        datasetData={datasetData}
+        dataset={dataset}
+        experiments={formattedExperiments}
         defaultOption={defaultOrganismFilterOption}
-        experiments={experiments}
         organism={organism}
         setOrganism={setOrganism}
       />
-      {experiments && (
+      {formattedExperiments && (
```

Comment:
I’ve removed `helpers/formatExperiments` from the codebase, and now format the experiments directly using `dataset.experiments` within the presentation components.

Additionally, I’ve renamed variables that were using the term `species` to `organism` in `SpeciesView` to align with the API terminology instead of the UI, as discussed during our 1:1.